redbot/__main__.py

```diff
@@ -391,6 +391,13 @@ async def run_bot(red: Red, cli_flags: Namespace) -> None:
                 print("Token has been reset.")
                 sys.exit(0)
         sys.exit(1)
+    except discord.PrivilegedIntentsRequired:
```

Comment:
This error is raised if I'm missing one of the intents in the portal and haven't used the flag...
the exact message is:

```
(Red-DiscordBot-QA) C:\Users\guyre\PycharmProjects\Red-DiscordBot>redbot intents --dev
c:\users\guyre\.virtualenvs\red-discordbot-qa\lib\site-packages\discord\client.py:250: DeprecationWarning: fetch_offline_members is deprecated, use chunk_guilds_at_startup instead
  self._connection = self._get_state(**options)
Red requires all Privileged Intents to be enabled.
You can find out how to enable Privileged Intents with this guide:
https://docs.discord.red/en/stable/bot_application_guide.html#enabling-privileged-intents
[2020-10-21 14:45:03] [INFO] red.main: Shutting down with exit code: 1
[2020-10-21 14:45:03] [INFO] red.main: Please wait, cleaning up a bit more

```
.... We should direct them to the new flag, or allow red to boot up without presence. at least imo... presense was sparcelly given to folks.

Reply:
The printed link explains everything needed already (just change `stable` to `latest` when using it for now, because we haven't released yet). As for directing to the flag, I think it will only cause us trouble with no benefit - it is already shown in `redbot --help` and since it is currently not supported, we shouldn't really advertise it more than that.